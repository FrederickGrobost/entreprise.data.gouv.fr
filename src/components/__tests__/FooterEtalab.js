import { createLocalVue, mount, RouterLinkStub } from "@vue/test-utils";
import { __createMocks as createStoreMocks } from "@/store/index.js";
import Vuex from "vuex";
import FooterEtalab from "@/components/FooterEtalab";

const localVue = createLocalVue();
localVue.use(Vuex);
localVue.component("router-link", RouterLinkStub);
jest.mock("@/store/index.js");

describe("FooterEtalab.vue snapshot testing", () => {
  const storeMocks = createStoreMocks();
  const wrapperFooter = mount(FooterEtalab, {
    localVue,
    store: storeMocks.store
  });

  test("It match the snapshot", () => {
    expect(wrapperFooter.vm.$el).toMatchSnapshot();
  });
});
