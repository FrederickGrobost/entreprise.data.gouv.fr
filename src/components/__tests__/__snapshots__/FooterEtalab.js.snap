// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterEtalab.vue snapshot testing It match the snapshot 1`] = `
<footer
  class="footer"
>
  <div
    class="container"
  >
    <div
      class="footer__logo"
    >
      <a
        href="https://www.etalab.gouv.fr"
        target="_blank"
      >
        <img
          src="../assets/img/etalab.svg"
        />
      </a>
       
      <ul
        class="footer__social"
      >
        <li>
          <a
            href="https://www.twitter.com/etalab"
          >
            <svg
              class="icon icon-twitter"
            >
              <use
                xlink:href="#icon-twitter"
              />
            </svg>
          </a>
        </li>
         
        <li>
          <a
            href="https://www.github.com/etalab"
          >
            <svg
              class="icon icon-github"
            >
              <use
                xlink:href="#icon-github"
              />
            </svg>
          </a>
        </li>
         
        <li>
          <a
            href="https://www.facebook.com/etalab"
          >
            <svg
              class="icon icon-fb"
            >
              <use
                xlink:href="#icon-fb"
              />
            </svg>
          </a>
        </li>
         
        <li>
          <a
            href="mailto:info@data.gouv.fr"
          >
            <svg
              class="icon icon-mail"
            >
              <use
                xlink:href="#icon-mail"
              />
            </svg>
          </a>
        </li>
      </ul>
    </div>
     
    <ul
      class="footer__links"
    >
      <li>
        <h2
          class="fit-mobile"
        >
          <a
            href="https://entreprise.data.gouv.fr/"
          >
            entreprise.data.gouv.fr
          </a>
        </h2>
      </li>
       
      <li>
        <a
          href="https://www.data.gouv.fr/fr/datasets/base-sirene-des-entreprises-et-de-leurs-etablissements-siren-siret/"
        >
          Données de référence
        </a>
      </li>
       
      <li>
        <a>
          Mentions légales
        </a>
      </li>
       
      <li>
        <a
          href="https://github.com/etalab/sirene_as_api"
        >
          API
        </a>
      </li>
       
      <li>
        <a
          href="https://status.entreprise.data.gouv.fr"
        >
          Statut des API
        </a>
      </li>
    </ul>
  </div>
</footer>
`;
